vendors/laiier/models/severn_wld/model.yaml

```
@@ -0,0 +1,49 @@
+# Commercial name of the model
+name: Severn WLD™
+# Functional description of the product. Maximum 500 characters.
+description: Severn WLD™ is a wireless smart LoRaWAN water leak detection system using an adhesive, printed sensor.
+# Logo of the device
+logo: Severn_wld.jpg
```

as described in the template, lower case and no underscore. 
```suggestion
logo: severn-wld.jpg
```